libs/api-bundle/src/Security/ManageApiToken/ManageApiTokenAuthenticator.php

```diff
@@ -47,6 +47,14 @@ public function authorizeToken(AuthAttributeInterface $authAttribute, TokenInter
         assert($authAttribute instanceof ManageApiTokenAuth);
         assert($token instanceof ManageApiToken);
 
+        if ($authAttribute->isSuperAdmin === false && $token->isSuperAdmin()) {
+            throw new AccessDeniedException('Authentication token must not be super admin');
+        }
+
+        if ($authAttribute->isSuperAdmin === true && !$token->isSuperAdmin()) {
+            throw new AccessDeniedException('Authentication token is not super admin');
+        }
```

Comment:
Takovy vtipny, jak `$authAttribute->isSuperAdmin` porovnavas explicitne s `true/false` a `$token->isSuperAdmin()` tam mas implicitne :smile:

Reply:
Tak u `$authAttribute->isSuperAdmin` je to potřeba protože je teď nullable. Ale upravím to ať je to stejný :D